frontend/src/app/main/ui/components/reorder_handler.scss

```diff
@@ -10,7 +10,7 @@
   flex-direction: column;
   height: calc(100% + var(--sp-m));
   justify-content: center;
-  left: var(--reorder-left-position, calc(-1 * var(--sp-l)));
+  left: var(--reorder-left-position, -11px);
```

Comment:
you can use logical-properties as well for these properties: `inset-inline-start`